### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
Added Popup windows for projects section

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,20 +1,130 @@
...
 const menu = ReadElement('#menu-icon');
 const nav = ReadElement('.mobile-nav');
 const closeNav = ReadElement('.close-button');
 const choose = ReadElement('.mobile-nav');
-
+const viewModal1 = ReadElement('.view-card-1');
+const viewModal2 = ReadElement('.view-card-2');
+const viewModal3 = ReadElement('.view-card-3');
+const viewModal4 = ReadElement('.view-card-4');
+const CardModal = ReadElement('.modal-section');
```
Comment (OyePriscilla) [inline index.js]: - [ ] Please let's be consistent according to JS best practises, this constant variable name suppose to be in camelCase I think. 👍

File: style.css [hunk 2]
```diff
@@ -607,6 +602,183 @@ textarea::placeholder {
...
+  min-height: 100%;
+  transition: 0.2s ease-in-out;
+  background-color: rgba(165, 185, 224, 0.5);
+  -webkit-backdrop-filter: blur(5px);
+  margin-bottom: 50px;
+  backdrop-filter: blur(5px);
+  display: none;
+  overflow: auto;
+  align-items: center;
+}
```
Comment (OyePriscilla) [inline style.css]: - [ ] It's awesome the way you rendered your POP details dynamically but lets quickly address these few issues that has to do with the given template. Designing a given template perfectly shows how well we pay attention to details.

**For the desktop version,**
1. Kindly align the button elements properly with the tech stacks. I could see an horizontal element in the figma design but it's missing from your design. ℹ️ 
2. The whole pop container has a margin bottom. 👍 

## Figma Design
Untitled

## Your Design
Untitled

**For mobile version,** 
1. Kindly ensure that all the elements are contained in the pop container.
2. When I clicked on the see project, the pop-up displayed different images.
3. The mobile version is totally different from what we have in the figma design.

## Figma Design
Untitled

## Your Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Bellagirl-maker 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Use JavaScript to manipulate the DOM ✔️ 
Implement JavaScript events ✔️ 
Store data with JavaScript object ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [optional] Please kindly format your README file according to the given [template](https://github.com/microverseinc/readme-template). Most importantly, **the tech stacks** and **key features** headers. 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Bellagirl-maker) [reply index.js]: @OyePriscilla Please are you referring to the CardModa? Because all the other variable names are in camelCase

**Comment by Bellagirl-maker** [comment]
@OyePriscilla I have made all requested changes

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Bellagirl-maker , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

212385281-4797c58a-d358-423a-aa7c-832c22203865

 ## Highlights
✔No linter issues
✔Good use of github workflow

 ## Required Changes ♻️
Good job on applying some of requested changes by the previous reviewer , but there is still one that wasn't fixed ,please add a divider between list of technologies and the buttons in your popup windows 
student popup divider

## Optional suggestions
_ Kindly increase the horizontal padding of your popup buttons to make them look like in figma desing 
_ Also the size of the popup window is very big the image is covering the screen width and hight kindly make it smaller so the user can see all its content .
student popup size

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bellagirl-maker** [comment]
@souad988 I have applied requested as well as optional changes

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Bellagirl-maker , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**